include/object/interrupt.h

```diff
@@ -21,7 +21,6 @@ void invokeIRQHandler_SetIRQHandler(irq_t irq, cap_t cap, cte_t *slot);
 void invokeIRQHandler_ClearIRQHandler(irq_t irq);
 void deletingIRQHandler(irq_t irq);
 void deletedIRQHandler(irq_t irq);
-void handleInterrupt(irq_t irq);
+static inline void maybeHandleInterrupt(void);
```

Comment:
```suggestion
void maybeHandleInterrupt(void);
```

I'd keep the name the same or call it checkInterrupt and combine it with handleInterrupt, should be easy if `irqInvalid > maxIRQ`.

Reply:
I'd prefer that.